### ISSUE TITLE
Are there any plans to support e-hentai sites?

### ISSUE THREAD

**Issue by zd20041** [AlexandreSenpai/Enma#65]
Are there any plans to support e-hentai sites?

**Comment by AlexandreSenpai** [comment]
Yeah, sure. Actually Its already on development state with Mangadex

**Comment by zd20041** [comment]
Where are you from?

**Comment by AlexandreSenpai** [comment]
I'm from Brazil, why?

**Comment by Spyridion** [comment]
Anyone know if there is some documentation on where new site parsing can be added?

**Comment by AlexandreSenpai** [comment]
@Spyridion Hello, do you mean where's a guide on how to extend Enma with custom sources? If so: https://enma.gitbook.io/enma/documentation/extending/sources

**Comment by AlexandreSenpai** [comment]
Youre welcome. Any questions let me know

**Comment by AlexandreSenpai** [comment]
Still trying to support e-hentai. The previous release gives access to Mangadex only
